### PR TITLE
Max08

### PR DESCRIPTION
**Pull request by MassimoVilla**
This PR involves some corrections on previous MSS records after our last two meetings: no new description. While merging to JJ, some conflicts might potentially need to be resolved, since this branch was created before the previous branch Max07 was merged, and both partially affect the same mss records.
Thank you.

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethe36.xml [hunk 1]
```diff
@@ -161,8 +161,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     The numeral <foreign xml:lang="gez">፮</foreign> has a compressed shape and the open loop.
                                 </desc>                               
                                 <seg type="rubrication">
-                                    Rubricated are the number and titles of the psalms, the stichometry, indicated by the word <foreign xml:lang="gez">ቃሉ፡</foreign> 
-                                    followed by a numeral, and elements of the numerals and of the punctuation signs, including text dividers.
+                                    The number and titles of the psalms; the stichometry, indicated by the word <foreign xml:lang="gez">ቃሉ፡</foreign>
```
Comment (SusanneHummel) [inline OxfordBodleian/Juel-Jensen/BDLaethe36.xml:164]: Why here "the" number...
without in BDLaethe30 and  BDLaethe33

File: OxfordBodleian/Juel-Jensen/BDLaethe30.xml [hunk 2]
```diff
@@ -580,8 +581,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </collation>
                                 <condition key="good">                   
                                     The textblock is affected with humidity, especially towards the end of the manuscript.
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethe30.xml:583]: Affected by? What do you think, @Gnisci ?

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml [hunk 3]
```diff
@@ -214,9 +215,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     The sewing is badly damaged and has come loose in many places. It was repaired after the misplacement of some leaves, which led to severe 
                                     perturbations in the textual order (e.g. the original position of <locus from="19" to="20"/> was after <locus target="#13"/>; <locus target="#74"/> is misplaced). 
                                     Some leaves are most probably missing.
-                                    The text block is severely affected with humidity. Most of leaves are stained with water, especially in the upper part, e.g. <locus target="#10rv #17rv #19rv #29rv"/>.
-                                    The text is barely legible in some places due to faded ink, e.g. <locus target="#10v #30v #35r #74r"/>. The red ink is mostly faded out.
-                                    Tears repaired on <locus target="#36 #87 #89"/>.                                   
+                                    The text block is severely affected with humidity. Most of leaves are stained with water, especially in the upper part, e.g. <locus target="#10rv #17rv #29rv"/>.
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethe33.xml:218]: same as before

File: OxfordBodleian/Juel-Jensen/BDLaethe36.xml [hunk 4]
```diff
@@ -121,7 +121,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     The boards are missing. The lowermost sewing station is damaged.
                                     The textblock is incomplete: an undetermined number of leaves is missing with loss of text from Ps 70 onwards.
                                     The textblock is slightly affected with humidity, especially at the beginning of the manuscript. Water stains are attested in the outer or inner margins of
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethe36.xml:123]: Same as before

File: OxfordBodleian/Juel-Jensen/BDLaethe30.xml (hunk 2 above)
Comment (Gnisci) [reply OxfordBodleian/Juel-Jensen/BDLaethe30.xml:583]: I am also not sure about the verbe and the tense. Also I think this damage affects the whole parchment not just the text block, so the whole sentence sounds a bit odd to me. "Humidity appears to have damaged the parchment, especially towards the end of the manuscript" or "Many of the pages have been affected/damaged by humidity, especially those at the end of the manuscripts"?

File: OxfordBodleian/Juel-Jensen/BDLaethe36.xml (hunk 1 above)
Comment (Gnisci) [reply OxfordBodleian/Juel-Jensen/BDLaethe36.xml:164]: Should say. Rubrication: "the number and title of the psalms" or "the numbers and titles of the psalms" for consistency. 

As for the "the" before the list, we can choose to include or exclude it as long as its consistent, I think we had opted for a more schematic approach.

Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethe36.xml:164]: Changed into "numbers and titles of the psalms"

**Review by SusanneHummel — APPROVED** [review]

**Review by Gnisci — APPROVED** [review]

**Pull request merged**